bokeh/models/component.py

```diff
@@ -7,7 +7,9 @@
 
 @abstract
 class Component(Model):
-    """ A base class for all embeddable models, i.e. plots and widgets. """
+    """ A base class for all embeddable models, i.e. plots, layouts and widgets.
```

Comment:
I guess "embeddable" is not the right work here anymore. Maybe "for all DOM-level components, i.e. plots, layouts, and widgets"